main_routers/workshop_router.py

```
@@ -0,0 +1,1134 @@
+        subscribed_items = steamworks.Workshop.GetSubscribedItems()
+        logger.info(f'获取到 {len(subscribed_items)} 个订阅的创意工坊物品')
+        
+        items_info = []
+        
+        for item_id in subscribed_items:
+            try:
+                if isinstance(item_id, str):
+                    try:
+                        item_id = int(item_id)
+                    except ValueError:
+                        logger.error(f"无效的物品ID: {item_id}")
```

Ruff (0.14.8)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)